frontend/packages/console-app/src/components/quick-starts/controller/QuickStartFooter.tsx

```diff
@@ -47,7 +49,7 @@ const QuickStartFooter: React.FC<QuickStartFooterProps> = ({
   ]);
 
   return (
-    <div className="co-quick-start-footer">
+    <div className={footerClass}>
```

Comment:
`co-quick-start-footer` should always be present. Do not rely on the parent to pass in this class.
Concatenate the incoming class to this.